### DISCUSSION TITLE
Sugguest Flow-Lancher supports the new feature that users can conveniently switch a targeted windows.

### DISCUSSION THREAD

**Discussion by climber418** [Flow-Launcher/Flow.Launcher#1243] [Ideas]
As everyone knows, it is difficult and inconveniently switch to a target windows especially when there are too many opened windows in taskbar.
It may be a great feature for Flow-Lancher to support switching to a window by keyboard rather than mouse.

**Comment by taooceros** [comment]
Check windowswalk plugin
